modules/extension/xsd/xsd-fes/src/test/java/org/geotools/filter/v2_0/bindings/FilterTypeBindingTest.java

```diff
@@ -2,7 +2,7 @@
  *    GeoTools - The Open Source Java GIS Toolkit
  *    http://geotools.org
  *
- *    (C) 2002-2015, Open Source Geospatial Foundation (OSGeo)
+ *    (C) 2002-2016, Open Source Geospatial Foundation (OSGeo)
```

Comment:
And our new policy is that we no longer require copyright date updates. Just leave existing OSGeo headers as they are.  :-)